FirebaseDatabase/Tests/Integration/FIRDatabaseQueryTests.m

```diff
@@ -4468,4 +4468,67 @@ - (void)testGetSkipsPersistenceCacheWhenOnline {
   WAIT_FOR(done);
 }
 
+- (void)testGetTriggersListener {
+  FIRDatabase* db = [self databaseForURL:self.databaseURL name:@"fir-multi-shards"];
+
+  FIRDatabaseReference* ref = [db referenceWithPath:@"james"];
+  FIRDatabaseReference* children = [ref child:@"children"];
+
+  __block BOOL done = NO;
+
+  [children removeValueWithCompletionBlock:^(NSError* _Nullable error,
```

Comment:
You could use an `autoId` instead of `children` (via `[FTestHelpers getRandomNode]`). Then you won't have to do a delete.